backend/app/services/doctransform/job.py

```
@@ -0,0 +1,272 @@
+
+
+@retry(wait=wait_exponential(multiplier=5, min=5, max=10), stop=stop_after_attempt(3))
+def execute_job(
+    project_id: int,
+    job_id: str,
+    source_document_id: str,
+    transformer_name: str,
+    target_format: str,
+    task_id: str,
+    callback_url: str | None,
+    task_instance,
```

Ruff (0.14.5)

Unused function argument: `task_instance`

(ARG001)